app/src/main/java/com/example/bankoflunar/dataStore/room/deposit_db/DepositDetails.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.bankoflunar.dataStore.room.deposit_db
+
+import androidx.annotation.NonNull
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class DepositDetails (
+    @PrimaryKey(autoGenerate = true)
+    val id: Int = 0,
```

Comment:
This `id` should always match with the id that with come from the api. Hence no default value should be set, and definitely shouldn't be auto generated